packages/server/src/service/signup.ts

```diff
@@ -26,21 +28,28 @@ export const sendAndStoreSignup = async (
   const letter = new Letter(info, method, id, resendReasonAndOriginalDate)
   const pdfBuffer = await toPdfBuffer(letter.render('html'), await letter.form)
   const isResend = resendReasonAndOriginalDate !== undefined
+  // When performing signups for DIVERT_EMAIL we don't send emails or faxes
```

Comment:
This is wrong -- we use `DIVERT_FAX_NUMBER` and `DIVERT_EMAIL` separately.  Let's stick to using that logic.

I don't think we should be editing `signup.ts` very much for this.